Patches/Core/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -1187,7 +1187,7 @@
     </AmmoUser>
     <FireModes>
       <aimedBurstShotCount>10</aimedBurstShotCount>
-      <aiAimMode>Snapshot</aiAimMode>
+      <aiAimMode>SuppressFire</aiAimMode>
```

Comment:
This will make the HCB Centipede less susceptible to corner-peeking. Currently, the player can peek and fire before the centi can complete its warm-up period and, on Snapshot, breaking LOS will interrupt the warmup entirely. The end result being that the centi never actually fires--setting it to Suppress, the mechs won't interrupt their burst.

Reply:
While the argument made is fair, it's fixing one very specific problem. The end result is going to be Blasterpedes that pretty much are not able to hit anything at range, they might be more capable of dealing with corner-peaking pawns but that's not something that happens all the time. A player that is advanced enough to deal with centis simply switches to long range fighting while the one in the early to mid-game only suffers since one of the reliable ,although risky, ways of dispatching blasterpedes will be out of the cards for them.

Reply:
Another thing, if the change is going to be pushed, a consistency check on other modded HCB-esque weapons is expected. Alpha Mechs and VFE-Mechs are two of the mods with said weapons in them.

Reply:
Corner peaking  with medium range AT launchers is still the single biggest threat against them, they still supress the fuck out of everything at their max range.
When wehicles come in *for real*, *then* we probably should consider turning them into an ant-vehicle countermeasure with aimed bursts.